tuf/ngclient/updater.py

```diff
@@ -200,6 +208,10 @@ def download_target(
                 ) from e
 
             filepath = os.path.join(destination_directory, target_filepath)
```

Comment:
```suggestion
            filepath = os.path.join(destination_directory, targetinfo["filepath"])
```

Reply:
You are right with this suggestion.
That way I will always save the file without the hash prefix - exactly what we want.